src/App.js

```diff
@@ -0,0 +1,152 @@
+import { useState } from "react";
+
+function Square({ value, onSquareClick, squareColor }) {
+  return (
+    <button
+      className="square"
+      onClick={onSquareClick}
+      style={{
+        backgroundColor: squareColor ? "white" : "yellow"
+      }}
+    >
+      {value}
+    </button>
+  );
+}
+
+function Board({ xIsNext, squares, onPlay }) {
+  const winner = calculateWinner(squares);
+  console.log(winner);
+  const row = [];
+  let u = 0;
+
+  function handleClick(i) {
+    if (squares[i] || calculateWinner(squares)) {
+      return;
+    }
+    const nextSquares = squares.slice();
+    if (xIsNext) {
+      nextSquares[i] = "X";
+    } else {
+      nextSquares[i] = "O";
+    }
+    onPlay(nextSquares);
+  }
+
+  let status;
+  if (winner) {
+    status = "Winner: " + winner[0];
+  } else if (!squares.includes(null) & !winner) {
+    status = "Game Over: Draw";
+  } else {
+    status = "Next player: " + (xIsNext ? "X" : "O");
+  }
+
+  for (let t = 0; t < 3; t++) {
+    const col = [];
+    for (let s = 0; s < 3; s++) {
+      let defaultColor = true;
+      if (winner) {
+        if (winner[1].includes(u)) {
+          defaultColor = false;
+        }
+      }
+      col.push(
+        <Square
+          key={u}
```

Comment:
```suggestion
          key={s}
```
This is not impacting the performance or correctness in this case - but just as a suggestion/best practice: it's usually better to use a more meaningful key. For example, in this case `col` will be consist of `Square`s. You can just use `s` or `` `column-{s}` `` in this case - the `Square` with the key `0` will actually be the first column, so it's more representative.

Right now we're using `u` for both columns and rows - so we can probably just replace the key for rows with `t` as well.